docs/conf.py

```diff
@@ -57,7 +55,39 @@
 #
 html_theme = 'alabaster'
 
+# Theme options are theme-specific and customize the look and feel of a theme
+# further.  For a list of options available for each theme, see the
+# documentation.
+#
+html_theme_options = {
+    'github_user': 'greenbone',
+    'github_repo': 'hyperion',
+    'github_banner': True,
+    'show_powered_by': False,
+}
+
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
+
+# Custom sidebar templates, must be a dictionary that maps document names
+# to template names.
+#
+# The default sidebars (for documents that don't match any pattern) are
+# defined by theme itself.  Builtin themes are using these templates by
+# default: ``['localtoc.html', 'relations.html', 'sourcelink.html',
+# 'searchbox.html']``.
+#
+# html_sidebars = {}
+html_sidebars = {
+    '**': [
+        'sidebarlogo.html',
+        'localtoc.html',
+        'relations.html',
+        'searchbox.html',
+    ]
+}
+
+subprocess.run(["poetry", "run", "python", "../manage.py", "graphql_schema", "--schema", "selene.schema.schema", "--out", "schema.json"])
+graphdoc = subprocess.run(["graphdoc", "-s", "schema.json", "-f", "-o", "_build/html/schema"])
```

Comment:
It's better to move the created schema file into a build directory to not accidentally include it into the git repo. Moving it into the build allows git to ignore that file.
```suggestion
subprocess.run(["poetry", "run", "python", "../manage.py", "graphql_schema", "--schema", "selene.schema.schema", "--out", "_build/schema.json"])
graphdoc = subprocess.run(["graphdoc", "-s", "_build/schema.json", "-f", "-o", "_build/html/schema"])

```